### PR TITLE
[Lead space] -Patterns - Lead space content update

### PR DESCRIPTION
**Pull request by DragosRistici**
### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/patterns/lead-space.mdx [hunk 1]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
...
-| 320px | Functional or navigational pages |
-| 240px | Minimal Lead space with simple title |
+| Super – 640px | Brand or immersive experience centric pages |
+| Tall – 560px | Marketing oriented pages |
+| Medium – 480px | Functional or navigational pages |
+| Short – 320px | Minimal lead space with simple title |
+
+### Lead space centered
 
+Lead space centered does not have fixed height variants and it is oriented more towards resolving functional and practical problems. If you want to transmit a more marketing-oriented message, please use the left-aligned lead space option.
```
Comment (shixiedesign) [inline src/pages/patterns/lead-space.mdx:99]: Some suggestions... we are lacking use cases at the moment to rigorously differentiate the usage of left-aligned vs centered, but here's my gut feeling: 

> Lead space centered does not have fixed heights. It is ideal for objective statements, such as academic research titles, and problem solving clues, such as `Find a product` or `Let's trouble shoot`. If you want to transmit a more marketing-oriented message, please use the left-aligned lead space option.

**Review by shixiedesign — COMMENTED** [review]
Take these comments with a grain of salt! I'm holding my head because a lot of these info should come from Lila and Kirby. Would be good to have a designer interview still, to get them to talk about how they design these.

File: src/pages/patterns/lead-space.mdx [hunk 2]
```diff
@@ -40,7 +40,7 @@ When designed and implemented effectively, the Lead space can be used to engage
 
 Here are some of the elements that are commonly used and currently available. As we continue to build and offer additional options, we will share new designs on this page.
 
-![Anatomy of the lead space](./images/lead-space/LeadSpace_Left_Heading06_Anatomy.jpg)
+![Anatomy of the lead space](../../images/pattern/leadspace/leadspace_patterns_anatomy_dark_demo.jpg)
```
Comment (photodow) [inline src/pages/patterns/lead-space.mdx:43]: Seeing as this is the pattern and not the component should we explore adding tags, breadcrumbs, and maybe search?

File: src/pages/patterns/lead-space.mdx [hunk 3]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
 
 ## Height standards
 
-The size of a lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
+### Lead space left-aligned
```
Comment (photodow) [inline src/pages/patterns/lead-space.mdx:86]: Remove aligned I think

File: src/pages/patterns/lead-space.mdx [hunk 4]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
...
 | Height | Recommended usage |
 | :--- | :--- |
-| 560px and taller | Brand or immersive experience centric pages |
-| 480px, 400px | Branding and messaging centric pages |
-| 320px | Functional or navigational pages |
-| 240px | Minimal Lead space with simple title |
+| Super – 640px | Brand or immersive experience centric pages |
+| Tall – 560px | High level and messaging centric pages |
+| Medium – 480px | Functional or navigational pages |
+| Short – 320px | Minimal and utilitarian. This lead space does not allow a button group and short copy. |
```
Comment (photodow) [inline src/pages/patterns/lead-space.mdx:95]: Add a column for the name.

**Review by photodow — CHANGES_REQUESTED** [review]
Just some thoughts. Keep in mind the pattern world is not my strong suit here.

- [ ] I understand you didn't add this one, but I wonder if this is really the example we want to lead with.

image

- [ ] I also understand you didn't add this, but should we add more clarification on these, especially that third one? It's not clear just looking at it it's shifted over because of the background image.

image

- [ ] It feels like there is a lot of duplication across pattern and component. I wonder if there is a way we could focus/differenciate these two pages a little more.

Most of my concerns aren't with what you provided, and more of what was there before. Thanks for the contribution! We should sync up on this work this week since I'm doing functional specs this sprint.

File: src/pages/patterns/lead-space.mdx (hunk 2 above)
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx:43]: Agree. In the next update, we will have to.

File: src/pages/patterns/lead-space.mdx [hunk 5]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
...
+The size of a left-align lead space height usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out more on Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
 
 | Height | Recommended usage |
 | :--- | :--- |
-| 560px and taller | Brand or immersive experience centric pages |
-| 480px, 400px | Branding and messaging centric pages |
-| 320px | Functional or navigational pages |
-| 240px | Minimal Lead space with simple title |
+| Super – 640px | Brand or immersive experience centric pages |
+| Tall – 560px | Marketing oriented pages |
```
[the remark this answers is not in the document]
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx]: Adjustment made.

File: src/pages/patterns/lead-space.mdx [hunk 6]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
...
+| Medium – 480px | Functional or navigational pages |
+| Short – 320px | Minimal lead space with simple title |
```
[the remark this answers is not in the document]
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx]: Adjustment made.

File: src/pages/patterns/lead-space.mdx [hunk 7]
```diff
@@ -83,15 +83,20 @@ On IBM.com pages, there are two layout options&mdash;left-aligned and center-ali
 
 ## Height standards
 
-The size of a lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
+### Lead space left-aligned
+
+The size of a left-align lead space height usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out more on Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
```
[the remark this answers is not in the document]
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx]: Adjustment made.

File: src/pages/patterns/lead-space.mdx (hunk 1 above)
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx]: Adjustment made.

File: src/pages/patterns/lead-space.mdx (hunk 4 above)
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx]: Adjustment made.

File: src/pages/patterns/lead-space.mdx (hunk 3 above)
Comment (DragosRistici) [reply src/pages/patterns/lead-space.mdx:86]: After the overhaul.

**Review by JennySanchez — APPROVED** [review]
Thank you, Dragos.

**Pull request merged**